### ISSUE TITLE
paddlex使用api训练语义分割模型时，如何使用多gpu？

### ISSUE THREAD

**Issue by bbbxixixixi** [PaddlePaddle/PaddleX#646]
目前的训练方式是定义
train_dataset = pdx.datasets.SegDataset(
    data_dir=TRAIN_DIR,
    file_list=TRAIN_FILE,
    label_list=LABEL_FILE,
    transforms=train_transforms,
    shuffle=True)
然后使用api进行模型训练
model = pdx.seg.HRNet(
    num_classes=N_CLASSES,
    width=WIDTH,
    class_weight=weights, 
    input_channel=4
)
model.train(
        num_epochs=EPOCHS,
        train_dataset=train_dataset,
        train_batch_size=BATCH_SIZE, 
        eval_dataset=eval_dataset,
        learning_rate=LR,
        log_interval_steps=20,
        save_interval_epochs=SAVE_INTERVAL,
        save_dir=SAVE_DIR,
        pretrain_weights='/home/aistudio/HRNet_W64_C_pretrained',
        use_vdl=True)
以前是单卡，现在有多卡，需要改哪些实现多gpu训练？

**Comment by jiangjiajun** [comment]
参考这个训练示例代码 https://github.com/PaddlePaddle/PaddleX/blob/develop/tutorials/train/semantic_segmentation/deeplabv3p_xception65.py

设置环境变量，如下设置表示使用第1和第2张卡，注意下面的代码必须放在执行代码最初始
```
import os
os.environ['CUDA_VISIBLE_DEVICES'] = '0,1'
```

**Comment by bbbxixixixi** [comment]
> 参考这个训练示例代码 https://github.com/PaddlePaddle/PaddleX/blob/develop/tutorials/train/semantic_segmentation/deeplabv3p_xception65.py
> 
> 设置环境变量，如下设置表示使用第1和第2张卡，注意下面的代码必须放在执行代码最初始
> 
> ```
> import os
> os.environ['CUDA_VISIBLE_DEVICES'] = '0,1'
> ```

就只需要这么设置就行了吗，不需要类似nn.DataParallel(model)一类的？

**Comment by jiangjiajun** [comment]
不需要，PaddleX内部已经封装好了，根据这个环境变量，会自动进行多卡训练
